src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClientUtils.groovy

```diff
@@ -100,4 +101,29 @@ class SaagieClientUtils {
 		}
 		return null
 	}
+	
+	def static extractVariableConfigAndPackageFromExportedVariable( File exportedZipFolder ) {
+		Map extractedConfig = [
+				variables : [ : ],
```

Comment:
what is this syntax ?

Reply:
This is a way to declare Map specific for the groovy language, allow us to create type-like json